spec/api-browser-view-spec.js

```diff
@@ -30,6 +30,22 @@ describe('BrowserView module', () => {
     return closeWindow(w).then(() => { w = null })
   })
 
+  describe('BrowserView.destroy()', () => {
+    it('does not throw', () => {
+      view = new BrowserView()
+      view.destroy()
+    })
+  })
+
+  describe('BrowserView.isDestroyed()', () => {
+    it('does not throw', () => {
```

Comment:
does not throw?

This should be `returns correct destoryed status` or something

Reply:
@MarshallOfSound Fixed!